coralproject/talk
# add Google auth plugin

## What does this PR do?
This adds a new plugin, talk-plugin-google-auth, which allows users to sign up / sign in with Google oauth2. It's basically a copy of the facebook plugin with a search and replace done. We had a version using the old auth system that was a bit more complicated, but the new auth system makes it easier.

## How do I test this PR?

As a signed out visitor, sign in. Click on the Sign in with google button and complete the sign in process with Google. The whole thing should be more or less identical to the Facebook sign up process.
